tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/transaction/MicroTransactionTestCase.java

```diff
@@ -309,24 +280,24 @@ public void testSaveToDatabaseFailedInParticipant() throws IOException {
         assertFalse(initiatorState.committedFunctionCalled);
         assertFalse(initiatorState.localParticipantCommittedFunctionCalled);
         assertTrue(initiatorState.abortedFunctionCalled);
-        assertTrue(initiatorState.localParticipantAbortedFunctionCalled);
+//        assertTrue(initiatorState.localParticipantAbortedFunctionCalled);
```

Comment:
What is the reason for commenting out these lines? Shall we remove them if not required?

Reply:
It was because we have disabled triggering aborted function on a participant.
Removed.

Reply:
Please create a github task to track this.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/12257